doc/multi-o.md

```diff
@@ -0,0 +1,117 @@
+# Multi Orchestrator Setup
+
+This document describes the various ways an orchestrator could run multiple nodes and separate concerns. 
+
+## Unlocking Account and Connecting to Ethereum
+
+Unlocking your Ethereum account and connecting to a JSON-RPC provider are necessary steps to run the Livepeer node unless it's in standalone transcoder mode. 
+
+For brevity we'll exclude these flags in the examples by `<...ETH SETUP...>` but these should always be required. If no Ethereum account is available the Livepeer node will create one for you.
+
+```
+livepeer
+    -network <mainnet|rinkeby> \
+    -ethUrl <JSON_RPC_PROVIDER> \
+    -ethKeystorePath <PATH_TO_KEY> \
+    -ethAcctAddr <ETHEREUM_ADDRESS> \
+    -ethPassword <ETHEREUM_ACCOUNT_PASSWORD (string|path)> \
+    -ethController <CONTROLLER_CONTRACT_ADDRESS (required if '-network' not provided)>
+```
+
+## Single Orchestrator with Redeemer
+
+This allows an orchestrator running a single on-chain node to use a seperate Ethereum account to redeem winning tickets on-chain and pay for that transaction while the recipient is still orchestrator node's Ethereum address.
+
+The orchestrator node will still be responsible for initialising rounds and calling `reward`. 
+
+1. Start the Redeemer 
+
+```shell
+livepeer
+    <...ETH SETUP...\
+    -redeemer=true \ 
+    -httpAddr <REDEEMER_HTTP_ADDR (host):port \ 
+    -ethOrchAddr <ORCHESTRATOR_ON_CHAIN_ETH_ADDR (also the recipient address)>
+```
+
+2. Start the Orchestrator
+
+
+```shell
+livepeer
+    <...ETH SETUP...> \
+    -orchestrator=true -transcoder=true \
+    -initializeRound=true \
+    -httpAddr <ORCH_HTTP_ADDR (host):port \
+    -pricePerUnit <PRICE (wei/pixel if '-pixelsPerUnit' is not set)
+```
+
+
+## Blockchain Service Node with Multiple Orchestrators
+
+In this setup a node started with the keys for the on-chain registered address will be responsible for all transactions. 
+
+In order to use multiple orchestrators with this setup, the cluster of orchestrator nodes responsible for transcoding would have to be behind a load balancer, **the URI of the load balancer will be the on-chain registered Service URI.**
```

Comment:
Maybe elaborate on what forms this load balancer could take, since that's not immediately clear from the description here? FWIW, I think DNS is basically the easiest option right now, other than having the broadcaster take an explicit list via webhook or `-orchAddr`. For those interested in building their own LB, it might be worth linking to the networking documentation around GetOrchestrator/OrchestratorInfo (it's unlikely anyone will try to at this stage, but maybe just mention it for completeness).

Also, I think the bolded parts can be split into its own sentence.

Reply:
I decided to not include any specifics about that here due to the fact that anything other than simple load balancing between a cluster of nodes within reasonable proximity is still an open area of investigation. We currently only use an explicit list on our broadcasters, which is only an option in a BOT setup. 

I think we can go more into detail once we have a better sense of the specifics here and what would work well for livepeer and what wouldn't ?

Reply:
Yeah, that's reasonable. Should we still mention "load balancing" even though we don't necessarily have concrete guidance around that? Seems like that would invite more questions than it answers. Maybe something like,

"In order to select multiple orchestrator nodes from the same operator, the broadcaster needs to explicitly list each orchestrator, either via the `-orchAddr` or`-orchWebhookUrl` flags. Additional load balancing options are being developed."

DNS should also work reasonably well out-of-the-box, although I don't know if anyone has actually tried it yet. Should we mention that as well?

Reply:
While that would work for internal usage or a BOT setup I think it might be good to have these docs also be useful for Orchestrators.  I would avoid mentioning broadcasters here. 

I also feel like the current non-default selection is just an intermediate step that works for us but not necessarily for your average broadcaster node. 

I tried DNS load balancing with NGINX but only for https, it was only a single O but it should work quite the same for multiple O's. 

I added an annotation "e.g. NGINX" but I can do "e.g. DNS Load Balancing" to not be vendor specific if you want.

Reply:
replaced with "e.g DNS load balancing"